arm/Microsoft.ContainerService/managedClusters/readme.md

```diff
@@ -183,50 +183,47 @@ For available properties check <https://docs.microsoft.com/en-us/azure/templates
 
 ### Parameter Usage: `agentPools`
 
+Provide values for additional agent pools as needed.
```

Comment:
Can be removed as the parameters table automatically links to the agent pool readme

Reply:
Removed section